INSTALL.md

```diff
@@ -11,6 +11,26 @@ PyBitmessage can be run in either one of two ways:
 - from an installed
```

Comment:
- point to pre-built binaries: windows exes as download.bitmessage.org, appimages at appimage.bitmessage.org, dpkgs (forgot, maybe download.bitmessage.org)
- remove references to directly executing from source

Reply:
added pre-build binaries references & removed references to directly executing from source

Reply:
Updated